[2025-10-23T20:59:36.244+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-23T20:59:36.290+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:59:34.869866+00:00 [queued]>
[2025-10-23T20:59:36.306+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:59:34.869866+00:00 [queued]>
[2025-10-23T20:59:36.307+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-10-23T20:59:36.329+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-10-23 20:59:34.869866+00:00
[2025-10-23T20:59:36.346+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2445) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-23T20:59:36.353+0000] {standard_task_runner.py:72} INFO - Started process 2447 to run task
[2025-10-23T20:59:36.352+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v2', 'get_name', 'manual__2025-10-23T20:59:34.869866+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp5caxm656']
[2025-10-23T20:59:36.356+0000] {standard_task_runner.py:105} INFO - Job 55: Subtask get_name
[2025-10-23T20:59:36.474+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v2.get_name manual__2025-10-23T20:59:34.869866+00:00 [running]> on host c65f76f8d89c
[2025-10-23T20:59:36.614+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v2' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T20:59:34.869866+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-23T20:59:34.869866+00:00'
[2025-10-23T20:59:36.615+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-23T20:59:36.620+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Olovuddin', 'last_name': 'Ramiddinov'}
[2025-10-23T20:59:36.665+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-23T20:59:36.666+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v2, task_id=get_name, run_id=manual__2025-10-23T20:59:34.869866+00:00, execution_date=20251023T205934, start_date=20251023T205936, end_date=20251023T205936
[2025-10-23T20:59:36.696+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-23T20:59:36.706+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
